### PR TITLE
Removed dbg! macros and changed deprecated range code

### PR DESCRIPTION
**Pull request by CatpainBlack**

### CODE REVIEW

File: src/ast/eval.rs [hunk 1]
```diff
@@ -88,7 +88,7 @@ impl Evaluate for Node {
     fn eval_with(&self, scope: &Scope) -> EvaluationResult {
         match self.token {
             Token::Operator(ref operator) => {
-                dbg!(&self.args);
+                //dbg!(&self.args);
```
Comment (chmln) [inline src/ast/eval.rs]: this could simply be removed, like the other `dbg!`

File: .gitignore [hunk 2]
```diff
@@ -8,3 +8,4 @@ Cargo.lock
 
 # These are backup files generated by rustfmt
 **/*.rs.bk
+.idea/
```
Comment (chmln) [inline .gitignore]: let's not add ide-specific ignores

**Review by chmln — CHANGES_REQUESTED** [review]
Hey, thanks for submitting this PR! 
Looks good overall, just some minor changes needed.

File: src/ast/eval.rs (hunk 1 above)
Comment (CatpainBlack) [reply src/ast/eval.rs]: Fixed in my latest commit

File: .gitignore (hunk 2 above)
Comment (CatpainBlack) [reply .gitignore]: Fixed in my latest commit

**Pull request closed without merging**